src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -112,6 +112,7 @@ func (r *ClusterReconciler) Reconcile(
 		pki.OperatorClientCert(),
```

Comment:
this is getting really bad, I need to take a look at this

Reply:
Yeah, we could group them together in structs like "pkiInfo"

Reply:
Similarly for the services

Reply:
I did `go mod tidy` also. No sarama dep anymore.